chromium_src/chrome/browser/resources/side_panel/customize_chrome/app.html.ts.lit_mangler.ts

```diff
@@ -77,3 +77,24 @@ mangle(
   },
   (template) => template.text.includes('sp-heading'),
 )
+
+// Add Brave Midnight(Darker) theme option
+mangle(
+  (element: DocumentFragment) => {
+    const appearance = element.querySelector('#appearance')
+    if (!appearance) {
+      throw new Error('[Customize Chrome] #appearance is gone.')
+    }
+
+    appearance.insertAdjacentHTML(
+      'beforeend',
+      /* html */ `
+      <brave-darker-theme-toggle ?hidden="\${
+        // @ts-expect-error This property is added by the app.ts file.
```

Comment:
rather than doing this you can use interface merging

https://github.com/brave/brave-core/blob/master/docs/webui_overriding.md#lit-functions

Reply:
Awesome! We've already run into that. Nice